ArtOfIllusion/src/artofillusion/RotateViewTool.java

```diff
@@ -154,18 +160,18 @@ private void dragRotateTravelSpace(WidgetMouseEvent e, ViewerCanvas view)
 		dx = dragPoint.x-clickPoint.x;
 		dy = dragPoint.y-clickPoint.y;
 
-		if (controlDown && !e.isShiftDown())
+		if (mouseButtonTwo(e) && controlDown)
 		{
 			view.tilting = true;
 			tilt(e, view, clickPoint);
 			return;
 		}
-		else if (controlDown && e.isShiftDown())
+		else if (mouseButtonTwo(e))
 		{
 			rotateSpace(e, view, clickPoint);
 			return;
 		}
-		else if (!controlDown && e.isShiftDown())
+		else if (mouseButtonThree(e) && e.isShiftDown())
```

Comment:
```suggestion
		else if (!controlDown && e.isShiftDown())
```
No MB3 available here!

Sorry for the `!controlDown`, see my other comment below or below that. The code could be restructured so, the negative condition would not be needed, but now I don't trust it to work without it.

Reply:
got it.